### DISCUSSION TITLE
Import secure sheets option

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2753] [Suggestions]
I see an option to export Sheets, but not to import. My onboarding process would be expedited greatly if I could import sheets as well

**Comment by JaspalSuri** [comment]
Hi @commitmentbasedsmartcontract, thanks for the feedback. 

You can convert your existing spreadsheets to the CSV format (which might be imperfect, depending upon the size and contents of the spreadsheet), convert them into a file that Standard Notes would recognize by using [the Plaintext to Standard Notes converter](https://dashboard.standardnotes.com/tools), and then import the file via the **Account** menu of the desktop or web apps.

**Comment by ghost** [comment]
Thanks Jaspal, will give it a go.
